pom.xml

```diff
@@ -705,6 +705,12 @@
 
             <dependency>
                 <groupId>org.wildfly.core</groupId>
+                <artifactId>wildfly-core-testsuite-shared</artifactId>
+                <version>${project.version}</version>
+            </dependency>
+
```

Comment:
I'm not sure if adding the testsuite-shared dependency to the dependencyManagement in the root pom is an acceptable way to solve this problem, but it seemed to work locally. If it passes at least it gives us a clue to the cause.